app/code/community/WindsorCircle/Analytics/etc/config.xml

```diff
@@ -313,6 +313,10 @@ updated_at
 total_orders
 total_spent
 </customer_traits>
+                <product_properties>
+cost
```

Comment:
Note: the purpose of this PR has changed so that this is **IS** a whitelist now. We also **want** these values, however, so this file's changes are unchanged.